src/libraries/System.Private.CoreLib/src/System/Globalization/IcuLocaleData.cs

```diff
@@ -24,586 +24,584 @@ internal enum IcuLocaleDataParts
+            "swc-cd"u8 +  // swc, swc-cd
+            "syr-sy"u8 +  // syr, syr-sy
+            "ta-in"u8 +  // ta, ta-in
+            "ta-lk"u8 +  // ta-lk
+            "ta-my"u8 +  // ta-my
+            "ta-sg"u8 +  // ta-sg
+            "te-in"u8 +  // te, te-in
+            "teo-ke"u8 +  // teo, teo-ke
+            "teo-ug"u8 +  // teo-ug
+            "tg-cyrl-tj"u8 +  // tg, tg-cyrl, tg-cyrl-tj
+            "th-th"u8 +  // th, th-th
+            "ti-er"u8 +  // ti, ti-er
+            "ti-et"u8 +  // ti-et
+            "tig-er"u8 +  // tig, tig-er
+            "tk-tm"u8 +  // tk, tk-tm
+            "tn-bw"u8 +  // tn, tn-bw
+            "tn-za"u8 +  // tn-za
+            "to-to"u8 +  // to, to-to
+            "tr-cy"u8 +  // tr, tr-cy
+            "tr-tr"u8 +  // tr-tr
+            "ts-za"u8 +  // ts, ts-za
+            "tt-ru"u8 +  // tt, tt-ru
+            "twq-ne"u8 +  // twq, twq-ne
+            "tzm-arab-ma"u8 +  // tzm, tzm-arab, tzm-arab-ma
+            "tzm-latn-dz"u8 +  // tzm-latn, tzm-latn-dz
+            "tzm-latn-ma"u8 +  // tzm-latn-ma
+            "tzm-tfng-ma"u8 +  // tzm-tfng, tzm-tfng-ma
+            "ug-cn"u8 +  // ug, ug-cn
+            "uk-ua"u8 +  // uk, uk-ua
+            "ur-in"u8 +  // ur, ur-in
+            "ur-pk"u8 +  // ur-pk
+            "uz-arab-af"u8 +  // uz, uz-arab, uz-arab-af
+            "uz-cyrl-uz"u8 +  // uz-cyrl, uz-cyrl-uz
+            "uz-latn-uz"u8 +  // uz-latn, uz-latn-uz
+            "vai-latn-lr"u8 +  // vai, vai-latn, vai-latn-lr
+            "vai-vaii-lr"u8 +  // vai-vaii, vai-vaii-lr
+            "ve-za"u8 +  // ve, ve-za
+            "vi-vn"u8 +  // vi, vi-vn
+            "vo-001"u8 +  // vo, vo-001
+            "vun-tz"u8 +  // vun, vun-tz
+            "wae-ch"u8 +  // wae, wae-ch
+            "wal-et"u8 +  // wal, wal-et
+            "wo-sn"u8 +  // wo, wo-sn
+            "x-iv_mathan"u8 +  // x-iv_mathan
+            "xh-za"u8 +  // xh, xh-za
+            "xog-ug"u8 +  // xog, xog-ug
+            "yav-cm"u8 +  // yav, yav-cm
+            "yi-001"u8 +  // yi, yi-001
+            "yo-bj"u8 +  // yo, yo-bj
+            "yo-ng"u8 +  // yo-ng
+            "yue-hk"u8 +  // yue, yue-hk
+            "zgh-tfng-ma"u8 +  // zgh, zgh-tfng, zgh-tfng-ma
+            "zh-chs"u8 +  // zh, zh-chs
+            "zh-cht"u8 +  // zh-cht
+            "zh-cn_phoneb"u8 +  // zh-cn, zh-cn_phoneb
+            "zh-cn_stroke"u8 +  // zh-cn_stroke
+            "zh-hans-hk"u8 +  // zh-hans, zh-hans-hk
+            "zh-hans-mo"u8 +  // zh-hans-mo
+            "zh-hant"u8 +  // zh-hant
+            "zh-hk_radstr"u8 +  // zh-hk, zh-hk_radstr
+            "zh-mo_radstr"u8 +  // zh-mo, zh-mo_radstr
+            "zh-mo_stroke"u8 +  // zh-mo_stroke
+            "zh-sg_phoneb"u8 +  // zh-sg, zh-sg_phoneb
+            "zh-sg_stroke"u8 +  // zh-sg_stroke
+            "zh-tw_pronun"u8 +  // zh-tw, zh-tw_pronun
+            "zh-tw_radstr"u8 +  // zh-tw_radstr
+            "zu-za"u8;  // zu, zu-za
```

Comment:
would ThreeLetterWindowsLanguageName below benefit from this treatment as well?

Reply:
Sure. I will update the generator.

Reply:
Turned out generator doesn't have support for `ThreeLetterWindowsLanguageName`, I made an ad-hoc bash/sed script for transformation.